lib/Parser/RegexRuntime.h

```diff
@@ -453,17 +454,7 @@ namespace UnifiedRegex
 #endif
     };
 
-    struct HardFailMixin
-    {
-        bool canHardFail;
-
-        inline HardFailMixin(bool canHardFail) : canHardFail(canHardFail) {}
-
-#if ENABLE_REGEX_CONFIG_OPTIONS
-        void Print(DebugWriter* w, const char16* litbuf) const;
-#endif
-    };
-
+#pragma pack(push, 1)
```

Comment:
in fact, looks like everything from L283-L1512 is already 1 byte aligned, so aren't all the intermediate pushes and pops meaningless?

Reply:
I had some redundant ones. Removed now.